images/linux/scripts/installers/azpowershell.sh

```diff
@@ -15,6 +15,10 @@ else
     versions=$(jq -r '.azureModules[] | select(.name | contains("az")) | .versions[]' $toolset)
 fi
 
+# Update PowerShellGet to eliminate download errors
+pwsh -Command "Install-Module -Name PowerShellGet -Force"
```

Comment:
Why can't we use toolset?
https://github.com/actions/virtual-environments/blob/main/images/linux/toolsets/toolset-1804.json#L88

Reply:
Powershell modules installation using toolset performs on 294 string of packer template, but az modules are installed on 216. As far as I know PowerShellGet should be updated before the az installation to eliminate errors, may be I should move pwsh modules installation to the earlier stage?